ex5/ex5.sh

```diff
@@ -0,0 +1,26 @@
+# 1
+for letter in {A..F}; do touch "file${letter}.txt"; done
```

Comment:
This works, but you were expected to use the ".." subtitution like this: `touch File{A..F}.txt`